src/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import 'antd/dist/antd.css';
```

Comment:
```suggestion
import 'antd/dist/antd.min.css';
```
antd.min.css로 변경부탁드려요! antd.css는 warning 표시가 자꾸 나타나더라구요!

Reply:
넵 수정해두었습니다!!